ch03tests/05Mocks.ipynb

```diff
@@ -4,54 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>I like to have the definition, as our course teaches to no-native speakers. That helps to make the connections of what we are trying to do.</p><p> </p><p>Real world examples could be added towards the bottom of the notebook, as to guide the students to look more details about it. <a href="https://webaccess.berkeley.edu/ask-pecan/click-here" target="_blank">Meaningful  hyperlinks helps accessibility</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UCL/rsd-engineeringcourse/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>